staticcheck.conf

```diff
@@ -0,0 +1,2 @@
+checks = ["all"]
+dot_import_whitelist=["github.com/onsi/gomega", "github.com/onsi/gomega/gexec", "github.com/onsi/ginkgo/v2"]
```

Comment:
1) Why are we still using Ginkgo v1?
2) I thought that staticcheck would not complain about dot imports in test files, so I'm surprised it was needed. We don't do it in the main Cloud Service Broker repo: https://github.com/cloudfoundry/cloud-service-broker/blob/538380d5f2df45b3a104758c8ec2de9a7304ae22/staticcheck.conf#L1

Reply:
Hey @blgm, thank you very much for reviewing the PR. 
In the case of the static check rule, unfortunately we are using gomega in our acceptance-test that are not test files.
![image](https://user-images.githubusercontent.com/9041034/175039279-da6c134f-198a-423e-b0c4-f1e17ec44a9f.png)

As for your first question, I don't understand exactly what you mean. We are using v2. Do you want to have a short meeting?